numba/np/arrayobj.py

```diff
@@ -2982,7 +2982,7 @@ def _increment_indices(context, builder, ndim, shape, indices, end_flag=None,
         count = shape[dim]
         in_bounds = builder.icmp_signed('<', idx, count)
         with cgutils.if_likely(builder, in_bounds):
-            # New index is still in bounds
+            # New index is stilll in bounds
```

Comment:
```suggestion
            # New index is still in bounds
```

I think this is a typo of a typo.

Reply:
Yikes, sorry about that!